Changelog O—G (#48393)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Adds different bleeding types for different species.
Adds bleeding butchery action.
Autodoc installation programs added that almost guarantee flawless installation of supported bionics.

> Unhardcode default Scenario/Profession & Make random starts respect Scenario Blacklists
> Removed plant mutant calorie generation
> mapgen: allow placing active bomb items for better craters in Dark Skies Above
> Allow Blacklisting/Whitelisting monsters by their species.
> monsters: Monsters on patrol
> Remove the 'carrion spawn rate scaling factor' world config option

Reviewer: ```suggestion
Allows JSON to change the default Scenario and Profession.
Follows Scenario blacklists with random starts.
Makes plant mutants' photosynthesis more realistic.
Allows placing active items - such as bombs for better craters - during mapgen.
Allows mods to blacklist or whitelist monsters by their species.
Enables setting a "patrol route" for monsters.
Removes the obsolete 'carrion spawn rate scaling factor' world configuration option.
```
Should "Makes plant mutants' photosynthesis more realistic." go into Bugfixes?

Adds different bleeding types for different species.
Adds bleeding butchery action.
Autodoc installation programs added that almost guarantee flawless installation of supported bionics.

> Makes plant mutants' photosynthesis more realistic.

Author: ```suggestion
```

Adds large dermatik nest and tweaks dermatiks.
Adds a new layout for the island prison challenge scenario.
Adds tazer drones.

> Adds demo of the Cuteclysm tileset. A visual set loosely inspired by the aesthetics of 16-bit era JRPGs.

Reviewer: ```suggestion
Adds a demo of the Cuteclysm tileset, which is loosely inspired by the aesthetics of 16-bit Japanese RPGs.
```

Adds fast scrolling to many menus with long lists.
Adds responsive screen behavior to the new character menu for small terminal sizes.
Shows weariness and activity levels in sidebar panels.

> Warns before crafting an item you won't be able to wear.
> Adds warning to excessive long attack times.

Reviewer: ```suggestion
Warns before crafting armor or clothing you won't be able to wear.
Warns if attack times will be excessively long (e.g., using a full backpack as a weapon).
```

[Aftershock] Adds new laser and plasma pistols.
[Magiclysm] Adds three new Attunement spells (Frozen Winds, Auroral wave, and Immolate).
[Magiclysm] Adds Workshop Elves, Yule Cats, and Leprechauns.

> Aftershock: Formless Ruins are a new, low-risk, above ground dungeon
> Dark Skies Above reborn!

Reviewer: ```suggestion
[Aftershock] Adds Formless Ruins - a new low-risk, above-ground dungeon type.
[Dark Skies Above] DSA is reborn as an actively-developed, total-conversion mod.
```

Avoids creating starving characters from older templates.
Fixes furniture being 'destroyed' when grab-moving.
Fixes crashes from canceling reloading a container that is on the ground.
Fixes NPCs still being hungry - and unhappy about it - after eating from faction camp stores.

> Reigned in plant mutation photosynthesis ability to be more representative of real photosynthesis.

Reviewer: ```suggestion
Reined in plant mutation photosynthesis ability to be more representative of real photosynthesis.
```

Avoids creating starving characters from older templates.
Fixes furniture being 'destroyed' when grab-moving.
Fixes crashes from canceling reloading a container that is on the ground.

> Fixes NPCs still being hungry - and unhappy about it - after eating from faction camp stores.

Author: ```suggestion
Fixes NPCs still being hungry - and unhappy about it - after eating from faction camp stores.
Reigned in plant mutation photosynthesis ability to be more representative of real photosynthesis.
```

Avoids creating starving characters from older templates.
Fixes furniture being 'destroyed' when grab-moving.
Fixes crashes from canceling reloading a container that is on the ground.

> Makes NPCs now eat from faction camps using their stomachs, eliminating phantom hunger pangs.

Reviewer: ```suggestion
Fixes NPCs still being hungry - and unhappy about it - after eating from faction camp stores.
```

When NPCs are fleeing from explosions, skips considering routing through tiles without a floor.
Speeds up fungus-related code processing by 30%+.
Optimizes item migrations to decrease load times.

> Fixes bad performance on large screens with lots of text (especially overmap).

Reviewer: ```suggestion
Restores graphics speed for large screens with lots of text.
```
Since this PR didn't actually improve FPS over 0.E-3, as far as I can tell, but instead restores it to that level - should it go into Bugfixes, not Performance?

Improves scooter tile representation by adding scooter vehicle parts, which scooters spawn with.
Reduces file size of composed tilesheets from compose.py.
Implements C++ symbol demangling in crash reports on MacOS, MinGW, and Linux.

> Adds tests of the various type of automatic trigger for mutations.
> Adds gun variants feature for definition of guns with only cosmetic differences.
> Add effect_on_conditions allowing use of dialog conditions/effects outside of dialog.

Reviewer: ```suggestion
Adds tests of automatic triggers for mutations.
Adds a gun variants feature, allowing easily defining multiple guns with only cosmetic differences.
Enables JSON-only specification of conditional effects.
```

Infrastructure: Automates translation template push.
Enables using additional languages (Arabic, Czech, Danish, Dutch, Greek, Indonesian, Icelandic, Italian, Norwegian, Serbian, Turkish, and Ukrainian) with the game.
Places cursor at highlighted item name in construction and crafting menus, as expected by screen readers.

> Translate Android installation UI.

Reviewer: ```suggestion
Adds translations of the Android installation interface.
```
